architecture-evaluation/src/main/java/edu/kit/kastel/sdq/case4lang/refactorlizar/architecture_evaluation/graphs/Node.java

```diff
@@ -1,53 +1,10 @@
 package edu.kit.kastel.sdq.case4lang.refactorlizar.architecture_evaluation.graphs;
 
-import java.util.Objects;
-import spoon.reflect.declaration.CtType;
-import spoon.reflect.declaration.CtTypeMember;
-import spoon.reflect.visitor.CtVisitor;
+/**
+ * @author Reiner Jung
+ * @param <T> module identifying type
+ */
+public interface Node<T> {
```

Comment:
warning: [MissingSummary] A summary line is required on public/protected Javadocs.
 * @author Reiner Jung
   ^
    (see https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)

Reply:
Also added a comment.